cartoframes/data/clients/data_obs_client.py

```diff
@@ -600,7 +600,7 @@ def augment(self, table_name, metadata, persist_as=None, how='the_geom'):
         if _meta.shape[0] == 0:
             raise ValueError('There are no valid metadata entries. Check '
                              'inputs.')
-        elif _meta.shape[0] > 50:
+        if _meta.shape[0] > 50:
```

Comment:
What if we change the previous behavior without raising an error? 

Not important since we are going to deprecate it